cpu/sam0_common/include/exti_config.h

```diff
@@ -0,0 +1,705 @@
+/*
+ * Copyright (C) 2020 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_sam0_common
+ * @brief           Generic EXTI map for all CPUs of the sam0 family.
+ * @{
+ *
+ * @author          Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ * @author          Derek Hageman <hageman@inthat.cloud>
+ */
+
+#ifndef EXTI_CONFIG_H
+#define EXTI_CONFIG_H
+
+#include "cpu_conf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*
+ * Used in the ASF headers, but not always defined by them (looks like they
+ * sometimes define __L instead).
+ */
+#ifndef _L
+#define _L(i) i ## L
+#endif
```

Comment:
Is it really needed ?

Reply:
Nope, looks like with the updated vendor files this is not needed anymore.